static/app/components/deprecatedforms/textareaField.tsx

```diff
@@ -23,4 +23,11 @@ export default class TextareaField extends InputField<InputField['props'], State
       />
     );
   }
+
+  /**
+   * Unused, only defined to satisfy abstract class
+   */
+  getType() {
+    return '';
```

Comment:
if it's really unused, I would throw a "Not implemented" error _just in case_ someone uses it in the future

Reply:
i think that might violate the typescript type if it doesn't return a string

Reply:
people seem to only use it in the getField, it seems to mostly serve no purpose https://github.com/getsentry/sentry/blob/5667aedffb98f77c1fff9fbc973ff7f052129326/static/app/components/deprecatedforms/booleanField.tsx#L32